.golangci.yaml

```
@@ -102,49 +154,43 @@ linters:
 
 issues:
   exclude:
-    - 'declaration of "(err|ctx)" shadows declaration at'
-    - "unnamedResult: consider giving a name to these results"
-    - "(Expect directory permissions to be 0750 or less|Expect (WriteFile|file) permissions to be 0600 or less)"
-
+    - declaration of "(err|ctx)" shadows declaration at
+    - 'unnamedResult: consider giving a name to these results'
+    - >-
+      (Expect directory permissions to be 0750 or less|Expect (WriteFile|file)
+      permissions to be 0600 or less)
   exclude-rules:
+    - linters:
+        - gosec
+      text: 'G602:'
     - path: _test\.go
       linters:
         - dupl
         - gocritic
         - funlen
-
     # https://github.com/go-critic/go-critic/issues/926
     - linters:
         - gocritic
-      text: "unnecessaryDefer:"
-
+      text: 'unnecessaryDefer:'
     - linters:
         - revive
-      text: "that stutters"
-
+      text: that stutters
     - path: zz_.*\.go
       linters:
         - revive
-      text: "it will be inferred from"
-
-    # Ignore the deprecated function use (staticcheck, SA1019) for the test files
+      text: it will be inferred from
     - path: (.+)_test.go
       linters:
         - staticcheck
-      text: "SA1019:"
-
+      text: 'SA1019:'
   include:
-    - EXC0011 # disable excluding of issues about missing package comments from stylecheck
-
-  # Show all errors for all linters. Setting these to 0 disables limiting error reporting.
+    - EXC0011
```

Can we keep the comment as it is.
```suggestion
    - EXC0011 # disable excluding of issues about missing package comments from stylecheck
```